tools/scripts/features/frcnn/extract_features_frcnn.py

```diff
@@ -0,0 +1,256 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import argparse
+import copy
+import glob
+import os
+
+import numpy as np
+import torch
+
+from modeling_frcnn import GeneralizedRCNN
+from processing_image import Preprocess
+from utils import Config
+
+
+class FeatureExtractor:
+
+    MODEL_URL = {"LXMERT": "unc-nlp/frcnn-vg-finetuned"}
```

Comment:
You can download the model and use it directly, that will allow us to get rid of lot of the model download logic.

Reply:
I have it downloaded and I pass it in as a command line argument. However the code seems to find the path to the model after it is first downloaded, should I still leave this for first time users to use?

Basically if you have nothing downloaded and you first run this it will download the model and store it somewhere, the next time you run it it just uses that cached model.

Reply:
Yeah we should check where in the code it is trying to donwload for first time users. Basically what I am saying is if we anyway package this model and config and provide it to users they don't need the download functionality. But if the code is calling it from somewhere we need to check from where and remove that call.